draft-ietf-rats-eat.md

```diff
@@ -818,6 +818,50 @@ uptime-claim = (
 )
 ~~~~
 
+## The Context Claim (context)
+
+EAT's may be used in the context of several different applications.  The context
+claim provides an indication to an EAT consumer about  the intended usage
+of the token.
+
+1 -- On-demand 
```

Comment:
Seems like this would be better named "generic". It's the one you pick when it's none of the others. I don't think it is on-demand because there will be cases when it is not one of the others and the consumer is not demanding the most recent.

Reply:
Updated draft to change to 'on-demand' to 'generic'